fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckAction.kt

```diff
@@ -26,6 +27,11 @@ sealed interface ReviewQualityCheckAction : Action {
      */
     sealed interface PreferencesMiddlewareAction : MiddlewareAction
 
+    /**
```

Comment:
does it make sense to group this above with the other `ReviewQualityCheckAction` interfaces?

Reply:
Hmm yeah, removed `MiddlewareAction` now that we're not using it. and the other 3 are grouped together